### ISSUE TITLE
Fetch blocked by CORS policy

### ISSUE THREAD

**Issue by reinisg** [developit/redaxios#74]
Hello,
this redaxios is great. We replaced axios with redaxios, globally all working fine (GET, POST, DELETE methods).
But there appeared new issue with PATCH. [Here is console error](https://i.ibb.co/RhQ4GsL/2021-11-13-093428.png).
This issue not appearing if switch back to axios.

What actually we need to prepare some change from server or only some fixes in request? 

Regards, Reinis

**Comment by donlion** [comment]
I've recently encountered an issue similar to this.
However the issue wasn't with redaxios but with Laravel.
Due to some technicalities with Laravel (I'm not exactly sure what) we've had to "patch the PATCH method" ourselves, and in doing this the server would look for `PATCH` not `patch` (case sensitive). The fix for me was to uppercase the HTTP method always
